packages/langium-cli/test/generator/types-generator.test.ts

```diff
@@ -15,7 +15,11 @@ describe('Types generator', () => {
     test('should generate types file', async () => {
         const result = (await parseHelper<Grammar>(grammar)(TEST_GRAMMAR)).parseResult;
         const typesFileContent = generateTypesFile(grammar, [result.value]);
```

Comment:
Suggestion: I believe it'd be better to just always remove `\r` from the generated results. Makes it less verbose (though at the cost of making it less explicit why we perform the replacement.

```suggestion
        const typesFileContent = generateTypesFile(grammar, [result.value]).replace(/\r/g, '');
```

Reply:
This is actually the better idea. This way the expectation is not changed. I will add a comment to why this is done.